aceptadora.go

```diff
@@ -57,9 +56,8 @@ func New(t *testing.T, imagePuller ImagePuller, cfg Config) *Aceptadora {
 // happening when most of the context has been consumed by pulling the image
 func (a *Aceptadora) PullImages(ctx context.Context) {
 	for svcName, svc := range a.yaml.Services {
-		t0 := time.Now()
+		a.t.Logf("Pulling image %q for %q", svc.Image, svcName)
```

Comment:
Isn't this log contradictory now? It may say _pulling_ when we're skipping the pull.

Reply:
I saw it as _starting the process of pulling_. But you're right. Removed in 635302d598b565d611ea625c1ce0b7abc404f7a1.